src/Themes/NexusMods.Themes.NexusFluentDark/Resources/ControlThemes/StandardButtonControlTheme.axaml

```diff
@@ -0,0 +1,457 @@
+                                      VerticalAlignment="{TemplateBinding VerticalAlignment}" />
+
+                    <!-- this is used when Content isn't set (used for normal buttons) -->
+                    <Border x:Name="PART_Border"
+                            Background="{TemplateBinding Background}"
+                            BorderBrush="{TemplateBinding BorderBrush}"
+                            BorderThickness="{TemplateBinding BorderThickness}"
+                            CornerRadius="{TemplateBinding CornerRadius}"
+                            Padding="{TemplateBinding Padding}">
+                        <StackPanel Orientation="Horizontal" VerticalAlignment="Center" HorizontalAlignment="Center">
+                            <icons:UnifiedIcon Name="PART_LeftIcon" />
+                            <TextBlock Name="PART_Label" Text="{TemplateBinding Text}" />
+                            <icons:UnifiedIcon Name="PART_RightIcon" />
+                        </StackPanel>
+                    </Border>
+                </Panel>
+            </ControlTemplate>
+        </Setter>
+
+        <!-- Set the default avalonia button styles -->
+        <Style Selector="^:pointerover">
+            <Setter Property="Background" Value="{DynamicResource ButtonBackgroundPointerOver}" />
+            <Setter Property="BorderBrush" Value="{DynamicResource ButtonBorderBrushPointerOver}" />
+            <Setter Property="Foreground" Value="{DynamicResource ButtonForegroundPointerOver}" />
+        </Style>
+
+        <Style Selector="^:pressed">
+            <Setter Property="Background" Value="{DynamicResource ButtonBackgroundPressed}" />
+            <Setter Property="BorderBrush" Value="{DynamicResource ButtonBorderBrushPressed}" />
+            <Setter Property="Foreground" Value="{DynamicResource ButtonForegroundPressed}" />
+        </Style>
+
+        <Style Selector="^:disabled">
+            <Setter Property="Background" Value="{DynamicResource ButtonBackgroundDisabled}" />
+            <Setter Property="BorderBrush" Value="{DynamicResource ButtonBorderBrushDisabled}" />
+            <Setter Property="Foreground" Value="{DynamicResource ButtonForegroundDisabled}" />
+        </Style>
+        <!-- end of default button styles -->
+
+
+        <!-- start of our new default button styles -->
+        <!-- these can also be overriden in NexusMods.Themes.NexusFluentDark -->
+
+        <!-- all button states -->
+        <Style Selector="^">
+            <Setter Property="ClipToBounds" Value="False" />
+            <Setter Property="Margin" Value="0" />
+            <Setter Property="Padding" Value="{DynamicResource ButtonPaddingMedium}" />
+            <Setter Property="CornerRadius" Value="{StaticResource Rounded}" />
+            <!-- Set an invisible border thickness to 1 to avoid size differences with outlined variants. -->
+            <Setter Property="BorderThickness" Value="0" />
```

Comment:
I think this might be an outdated comment? We are not doing it and I don't see a difference in size.

I think I saw this problem in rounded buttons specifically before, where the extra border could shift the contents a bit, but with more squared design it should not be a problem I think.